cpp/ql/src/Security/CWE/CWE-022/TaintedPath.qhelp

```diff
@@ -41,6 +62,7 @@ access all the system's passwords.</p>
 OWASP:
 <a href="https://owasp.org/www-community/attacks/Path_Traversal">Path Traversal</a>.
 </li>
+<li>Rails: <a href="https://api.rubyonrails.org/classes/ActiveStorage/Filename.html#method-i-sanitized">ActiveStorage::Filename#sanitized</a>.</li>
```

Comment:
We should change this to a C/C++ link.  The best I can find is https://man7.org/linux/man-pages/man3/realpath.3.html , though I don't really like that function.

Reply:
It's also got a horrible interface.  But it's better than no suggestion at all.  :+1: